compiler/ghc/default.nix

```
@@ -41,7 +41,7 @@
 , # What flavour to build. An empty string indicates no
   # specific flavour and falls back to ghc default values.
   ghcFlavour ? stdenv.lib.optionalString (stdenv.targetPlatform != stdenv.hostPlatform)
-    (if useLLVM then "perf-cross" else "perf-cross-ncg")
+    (if useLLVM then "quick-cross" else "quick-cross-ncg")
```

I think I'd prefer to keep `perf-cross-ncg`, the only current use for that is windows. I'd prefer to keep the `perf` version if we can for release builds. Maybe we should add a parameter to the default.nix to trigger quick/perf builds?

---

We could do that, but the aarch32 build is going to need the `quick` flavor, unless we can debug why QEMU crashes on executables built with a `perf-cross` compiler.

I could just add an extra conditional that checks for aarch32 and cross-compilation and makes `quick-cross` builds then?

---

I'm just suggesting this

```suggestion
    (if useLLVM then "quick-cross" else "perf-cross-ncg")
```

aarch32,... will all use LLVM. Windows will do the NCG instead of LLVM as we are targetting x86_64.